dist/angular-sticky-table-header.js

```diff
@@ -109,7 +109,8 @@ angular.module('turn/stickyTableHeader', ['watchDom']).value('stickyTableHeaderO
             } else if (scope.stuck && scrollY < scope.offset.top) {
               scope.setStuck(false);
             }
-            scope.clone.css('left', scope.offset.left - scrollX);
```

Comment:
@bcherny If we set left:0 from css directly then while scrolling left position is override by setCloneGutter() which is called prior to checkScroll() and  header left position is set to some value instead of 0.

Hence, we explicitly need to set left position to 0 in checkScroll().